update_and_render.sh

```diff
@@ -1,13 +1,24 @@
-# Fetch the latest rpc.proto
-curl -o rpc.proto -s https://raw.githubusercontent.com/lightningnetwork/lnd/master/lnrpc/rpc.proto
-# cp ~/lightning/ln-lnd/src/github.com/lightningnetwork/lnd/lnrpc/rpc.proto .
+# We'll default to fetching the latest version of the RPC files. Otherwise, we'll use the commit hash provided.
+commit="master"
+[ -n "$1" ] && commit=$1
 
-# Generate the rpc.json file from rpc.proto, so that generate_slate_docs.py can parse it
+curl -o rpc.proto -s https://raw.githubusercontent.com/lightningnetwork/lnd/$commit/lnrpc/rpc.proto
+curl -o rpc.swagger.json -s https://raw.githubusercontent.com/lightningnetwork/lnd/$commit/lnrpc/rpc.swagger.json
+
+# Generate the rpc.json file from rpc.proto, so that render.py can parse it.
 protoc -I. -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --doc_out=json,rpc.json:. rpc.proto
 
-# Generate the new docs
+# Update lncli to the respective commit.
+pushd $GOPATH/src/github.com/lightningnetwork/lnd
+git pull
+git reset --hard $commit
+make
```

Comment:
We don't need to make anything. We only care about the protos.

Reply:
We'll need it in order to provide updated documentation for the `lncli` shell commands.